websites/D/diziwatch/metadata.json

```
@@ -0,0 +1,20 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.10",
+	"author": {
+		"id": "510174958270545920",
+		"name": "emrovsky"
+	},
+	"service": "diziwatch",
+	"description": {
+		"en": "diziwatch is a platform you can watch animes,movies"
+	},
+	"url": "diziwatch.net",
```

There are new fields called `matches` and `iFrameMatches`, which are required for new Presences.
These are an array of strings which need to follow https://developer.chrome.com/docs/extensions/develop/concepts/match-patterns. 

```suggestion
	"url": "diziwatch.net",
	"matches": ["*://diziwatch.net/*"],
```